packages/create-plugin/templates/github/ci/.github/workflows/ci.yml

```diff
@@ -142,3 +114,107 @@ jobs:
           name: $\{{ steps.metadata.outputs.plugin-id }}-$\{{ steps.metadata.outputs.plugin-version }}
           path: $\{{ steps.metadata.outputs.plugin-id }}
           retention-days: 5
+
+  resolve-versions:
+    name: Resolve e2e images
+    runs-on: ubuntu-latest
+    timeout-minutes: 3
+    needs: build
+    if: $\{{ needs.build.outputs.has-e2e == 'true' }}
+    outputs:
+      matrix: $\{{ steps.resolve-versions.outputs.matrix }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Resolve Grafana E2E versions
+        id: resolve-versions
+        uses: grafana/plugin-actions/e2e-version@main
+
+  playwright-tests:
```

Comment:
does this need to have `if: $\{{ needs.build.outputs.has-e2e == 'true' }}` too? or would the previous job validation be enough?

Reply:
The condition is inferred, so not necessary. (tested [here](https://github.com/sunker/sunker-cicd-datasource/actions/runs/8721356526?pr=1) just to be sure)

![Screenshot 2024-04-17 at 14 01 32](https://github.com/grafana/plugin-tools/assets/2388950/48df708e-5231-4208-8cda-96689ca90079)

I wish there was a way to simply omit the job from the workflow summary in case it's being skipped. 🤔